com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolumeProfile.cs

```diff
@@ -11,29 +11,55 @@ namespace UnityEngine.Rendering
     /// </summary>
     public sealed class ProbeReferenceVolumeProfile : ScriptableObject
     {
-        /// <summary>
-        /// The default dimensions for APV's index data structure.
-        /// </summary>
-        public static Vector3Int s_DefaultIndexDimensions = new Vector3Int(1024, 64, 1024);
+        internal enum Version
+        {
+            Initial,
+        }
+
+        public enum CellSize
+        {
+            [InspectorName("1")]
+            CellSize1 = 1,
+            [InspectorName("9")]
+            CellSize9 = 9,
+            [InspectorName("27")]
+            CellSize27 = 27,
+            [InspectorName("81")]
+            CellSize81 = 81,
+            [InspectorName("243")]
+            CellSize243 = 243,
+        }
 
         /// <summary>
         /// The size of a Cell.
         /// </summary>
-        [Min(1)]
-        public int cellSize = 64;
+        public CellSize cellSizeInBricks = CellSize.CellSize81;
+
         /// <summary>
         /// The size of a Brick.
         /// </summary>
-        [Min(0.3333f)]
+        [Min(0.01f)]
         public float minDistanceBetweenProbes = 1.0f;
         /// <summary>
         /// The normal bias to apply during shading.
         /// </summary>
         [Range(0.0f, 1.0f), Delayed]
         public float normalBias = 0.2f;
 
-        public int maxSubdivision => Mathf.CeilToInt(Mathf.Log(cellSize / brickSize, 2));
-        public int brickSize => Mathf.Max(1, Mathf.RoundToInt(minDistanceBetweenProbes * 3.0f));
+        [SerializeField]
+        Version version = CoreUtils.GetLastEnumValue<Version>();
+
+        public int maxSubdivision => Mathf.CeilToInt(Mathf.Log((float)cellSize / brickSize, 3));
+        public float brickSize => Mathf.Max(0.01f, minDistanceBetweenProbes * 3.0f);
+        public int cellSize => Mathf.CeilToInt((float)cellSizeInBricks * brickSize);
+
+        void OnEnable()
+        {
+            if (version != CoreUtils.GetLastEnumValue<Version>())
+            {
+                // Migration code
```

Comment:
Planned TODO or is something forgot?

Reply:
It's in case we need to migrate some stuff one day (and I'm pretty sure it will happen :) )

Reply:
I mean we already have to with the cell size stuff, so probably should add something here if  we can.

Reply:
We can't, the problem is that there was no version field before, so it's not possible to detect that the asset was at a previous version, we had this issue several times already in HDRP.
So for this PR it will discard the value that was in cell size (breaking change) but it's fine since the feature isn't released yet.